experimental/segmenter/src/grapheme.rs

```diff
@@ -10,11 +10,16 @@ use crate::complex::{Dictionary, LstmPayloads};
 use crate::indices::{Latin1Indices, Utf16Indices};
 use crate::provider::*;
 use crate::rule_segmenter::*;
+use utf8_iter::Utf8CharIndices;
 
 /// Grapheme cluster break iterator for an `str` (a UTF-8 string).
 pub type GraphemeClusterBreakIteratorUtf8<'l, 's> =
     RuleBreakIterator<'l, 's, GraphemeClusterBreakTypeUtf8>;
 
+/// Grapheme cluster break iterator for a potentially invalid UTF-8 string.
+pub type GraphemeClusterBreakIteratorPotentiallyInvalidUtf8<'l, 's> =
```

Comment:
Nit:

```suggestion
pub type GraphemeClusterBreakIteratorPotentiallyIllFormedUtf8<'l, 's> =
```

CC @markusicu